addon/const/service-links.js

```diff
@@ -28,11 +28,11 @@ const serviceLinks = {
     preprintsDiscover: `${osfUrl}preprints/discover/`,
     preprintsHome: `${osfUrl}preprints/`,
     preprintsSubmit: `${osfUrl}preprints/submit/`,
-    preprintsSupport: 'http://help.osf.io/m/preprints/',
+    preprintsSupport: 'https://help.osf.io/hc/en-us/categories/360001530554-Preprints',
     profile: `${osfUrl}profile/`,
     registriesDiscover: `${osfUrl}registries/discover/`,
     registriesHome: `${osfUrl}registries/`,
-    registriesSupport: 'http://help.osf.io/m/registrations/',
+    registriesSupport: 'https://help.osf.io/hc/en-us/categories/360001550953-Registrations',
```

Comment:
Registries is actually fine, it isn't using this.

There are [some things](https://github.com/CenterForOpenScience/ember-osf-web/search?q=zendesk&unscoped_q=zendesk) that could be updated, but they seem to redirect properly when accessed.